assets/base.css

```diff
@@ -2161,7 +2161,7 @@ input[type='checkbox'] {
 .announcement-bar__link {
   display: block;
   width: 100%;
-  padding: 1rem 2rem;
+  padding: 1rem 0;
```

Comment:
My initial thought was combine both selectors and declare the padding just once like this:

```css
.announcement-bar__link,
.announcement-bar__message {
  padding: 1rem 0;
}
```

... but the way you have it now works perfectly and it'll make it easier if we ever needed to separately adjust `announcement-bar__link` or `announcement-bar__message` in the future.